src/rrr_riv_tot_scl_prm.py

```diff
@@ -20,6 +20,7 @@
 #*******************************************************************************
 import sys
 import csv
+import json
 
 
 #*******************************************************************************
```

Comment:
The description of the declaration of variables for arg 3 and arg 4 need to be updated to huc_8_map_k and huc_8_map_x